### DISCUSSION TITLE
Trestle Project

### DISCUSSION THREAD

**Discussion by AdrianHammond** [finos/compliant-financial-infrastructure#283] [General]
I had a discussions with Anca Sailer re the [Trestle Project](https://github.com/IBM/compliance-trestle). Trestle is an ensemble of tools that enable the creation, validation, and governance of documentation artefacts for compliance needs. Anca shared some links to following blogs and some slides

- https://dzone.com/articles/compass-compliance-part-1
- https://dzone.com/articles/compliance-automated-standard-solution-compass-part-2-trestle-sdk
- https://dzone.com/articles/compliance-automated-standard-solution-compass-part-3-artifacts-and-personas

[Trestle Compliance as Code System of Record_AncaSailer.pdf](https://github.com/finos/compliant-financial-infrastructure/files/9908686/Trestle.Compliance.as.Code.System.of.Record_AncaSailer.pdf)

My view is this is something that the Policy WG should review once it is established, as I believe that Trestle capabilities may be relevant in the future but it is probably beyond where we are with CFI today. 

@abdullahgarcia please could you take a look at share any thoughts. 

@eddie-knight @mcleo-d @thinkl33t @peter-thomas-db

**Comment by abdullahgarcia** [comment]
Taking a look at this tomorrow.
